src/com/sun/ts/tests/jaxws/jaxws23/wsa/j2w/document/literal/anonymous/AddNumbersService23001.wsdl

```diff
@@ -1,25 +1,8 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!--
-
-    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
-
```

Comment:
I noticed that we are deleting the copyright notice, should that be kept?

Reply:
Have added back the copyright, thanks for letting me know.